src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    if (!fixedRobot.hasOwnProperty(robot[key])) {
+      fixedRobot[robot[key]] = key;
+    } else {
+      return null;
+    }
```

Comment:
Greate practice is write `if`  wich break your function and then main code
```suggestion
    if (fixedRobot[robot[key]]){
      return;
    }
    
    fixedRobot[robot[key]] = key;
```